app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt

```diff
@@ -33,14 +35,16 @@ class NewCreatePinFragment :
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        presenter.pinMode = PinMode.CREATE
```

Comment:
no need to put explicitly put this value here, it has default in Presenter

Reply:
Actually is has to reset to default mode in case of returning back from next screen